<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TouchScreenCalibrator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="screenPicture">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>331</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/img/deksptop.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startCalibration">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(56, 84, 125);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressources.qrc">
      <normaloff>:/img/img/xinput_calibrator.png</normaloff>:/img/img/xinput_calibrator.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="yMinMinus">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(95, 141, 211);</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xMinMinus">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>50</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(95, 141, 211);</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="yMaxMore">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(95, 141, 211);</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xMaxMore">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>220</y>
      <width>50</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(95, 141, 211);</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="min_x">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="max_x">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>240</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="min_y">
    <property name="geometry">
     <rect>
      <x>458</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="max_y">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="applicationTitle">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(56, 84, 125);</string>
    </property>
    <property name="text">
     <string>TouchScreen Calibrator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="startCAlibrationLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(56, 84, 125);</string>
    </property>
    <property name="text">
     <string>Start  Calibration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yMinMore">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="right_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(95, 141, 211);</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xMaxMinus">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>50</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xMinMore">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>50</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="yMaxMinus">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>771</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>FontAwesome</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Device  : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="availableDevice">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>FontAwesome</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="allDevice">
       <property name="text">
        <string>All device</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="load">
       <property name="font">
        <font>
         <family>FontAwesome</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Load </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="font">
        <font>
         <family>FontAwesome</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Save </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="test">
       <property name="font">
        <font>
         <family>FontAwesome</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Test </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="apply">
       <property name="font">
        <font>
         <family>FontAwesome</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Apply </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>167</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="swapAxisLabel">
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string>color: rgb(56, 84, 125);</string>
       </property>
       <property name="text">
        <string>Swap Axis</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="invertX">
       <property name="font">
        <font>
         <family>FontAwesome</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(95, 141, 211);</string>
       </property>
       <property name="text">
        <string></string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="invertY">
       <property name="font">
        <font>
         <family>FontAwesome</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(95, 141, 211);</string>
       </property>
       <property name="text">
        <string></string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="swapAxis">
       <property name="font">
        <font>
         <family>FontAwesome</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(95, 141, 211);</string>
       </property>
       <property name="text">
        <string></string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="invertXLabel">
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(56, 84, 125);</string>
       </property>
       <property name="text">
        <string>Invert X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="invertYLabel">
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(56, 84, 125);</string>
       </property>
       <property name="text">
        <string>Invert Y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="thresholdLabel">
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(56, 84, 125);</string>
       </property>
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="threshold">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>A propos</string>
    </property>
   </widget>
   <addaction name="menuA_propos"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
